### DISCUSSION TITLE
Examples containing `yq r`

### DISCUSSION THREAD

**Discussion by rfs613** [mikefarah/yq#792] [Q&A] [answered]
In the docs [1] there are a number of examples which start with `yq r`.
These produce an error when trying to run:
```
$ yq r -j -P -I4 sample.yaml
Error: unknown command "r" for "yq"
Run 'yq --help' for usage.
```
Should the `r` be changed to `e`?

[1] https://mikefarah.gitbook.io/yq/usage/output-format

**Comment by mikefarah** [comment, marked as answer]
yes, yes it should. Thanks, I'll update the docs
